Userland/Libraries/LibWasm/Wasi.h

```diff
@@ -369,7 +370,8 @@ struct FSTFlags {
         bool atim_now : 1;
         bool mtim : 1;
         bool mtim_now : 1;
-        u16 _unused : 12;
+
+        u8 : 8;
```

Comment:
Wait, don't these changes in WASI turn unused fields into padding?

Reply:
Kinda, looks like no one uses `_unused`.
The alternative is to split `_unused` into ~8 fields in each case.

Reply:
No one is supposed to use `_unused`, it's explicit padding because host alignments are different.

Reply:
@alimpfard then what are your thoughts on truncating the unused bits here? It feels weird to say that the unused fields are 8/16 bits when in reality they are longer than that. We have the static_assert at the bottom... I'm still not sure what the actual impact is on initialization of these bitfields if we turn the extra bits from an `_unused` field into compiler-generated padding. I think it means that the bits are indeterminate, and we can't rely on ` = {}` to zero-initialize the padding anymore.

Reply:
:thonk: I'd like to avoid producing implicit paddings here, the layout is specified (even though it's done in a somewhat weird manner)

I suppose we could use 8 `u8 _unused_bit_1 : 1`?